src/main/resources/META-INF/neoforge.mods.toml

```diff
@@ -15,28 +15,28 @@ logoFile = "assets/${modId}/icon.png"
 [properties]
 "connector:placeholder" = true
 
-[[dependencies.${ modId }]]
+[[dependencies."${modId}"]]
 modId = "connector"
 type = "required"
 versionRange = "*"
 ordering = "NONE"
 side = "BOTH"
 
-[[dependencies.${ modId }]]
+[[dependencies."${modId}"]]
 modId = "neoforge"
 type = "required"
 versionRange = "*"
 ordering = "NONE"
 side = "BOTH"
 
-[[dependencies.${ modId }]]
+[[dependencies."${modId}"]]
 modId = "minecraft"
 type = "required"
 versionRange = "[${mc},)"
 ordering = "NONE"
 side = "BOTH"
 
-[[dependencies.${ modId }]]
```

Comment:
why's this a thing?

Reply:
i responded to the comment on the initial issue about this, i initially kept this tweak because that's how the default neoforge mod template does it, and intellij idea throws a warning if the spaces are present.

since it works fine anyways, rolled back the change for now.

Reply:
I'll put it in 1.21 instead and merge it over!